plugins/examples/pod-counter/src/index.tsx

```diff
@@ -9,11 +9,19 @@ function PodCounter() {
 
 registerAppBarAction(PodCounter);
 
-// registerAppBarAction({
-//   processor: (_, actions) => {
-//     const newActions = actions.filter(action => action.id !== 'pod-counter')
-//     newActions.push({action: <PodCounter />, id: 'pod-counter'})
-//     return newActions;
-//   },
-//   id: 'pod-counter-processor',
-// });
+// We can also reorder the actions in the app bar.
+registerAppBarAction((_, actions) => {
```

Comment:
You have previously asked me to name the functions in such examples, so it became clearer for the user. Should this one be `function reorderNotifications`.

Reply:
BTW, the reason why I made the header actions processor receive the `KubeObject` as first arg was that they're in the context of a details view which will have an object like that.
I think that if we never pass the 1st arg to the app bar processor, then we should remove that first arg.

Reply:
> BTW, the reason why I made the header actions processor receive the KubeObject as first arg was that they're in the context of a details view which will have an object like that.
I think that if we never pass the 1st arg to the app bar processor, then we should remove that first arg.

I'd like to rethink the processor input so it can take a dictionary.

```
registerAppBarAction(({ actions }) => {
```

So it can be extended later. 

I can imagine people wanting different buttons depending on the resource.

But I think that can come in a different refactor?